facebook/app/views/posts/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<p id="notice"><%= notice %></p>
+
+<h1>Posts</h1>
+<%= link_to "Home", root_path %>
+<table>
+  <thead>
+    <tr>
+      <th>Content</th>
+      <th colspan="3"></th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @posts.each do |post| %>
```

Comment:
- [ ] Post is always displayed with content and author -- let's add the author. 
We need to apply the same requirement in `show.html.erb` for posts.